spec/controllers/users_controller_spec.rb

```diff
@@ -30,10 +43,24 @@
     end
   end
 
+  context "GET #show" do
+    context "when not logged in" do
+      it "won't access the show page" do
+        user = create(:user)
```

Comment:
Also here, no need to create a user. The before action should happen before it even tries to find a user.

Reply:
I was thinking I would need the user.id but any random does the job, thanks, done in sha: 08e222e